### PR TITLE
add tf var github org and token references to platform workflows

### PR DESCRIPTION
**Pull request by bbleau**

### CODE REVIEW

File: .github/workflows/platform-check.yaml [hunk 1]
```diff
@@ -4,6 +4,13 @@
 # repo then mentioned above as they will be overwritten upon the
 # next sync.
 
+# NOTE: we are temporarily throwing the kitchen sink of variables
+# at the workflow until we come up with a better solution at a
+# future date.
+env:
+  TF_VAR_GITHUB_ORG: ${{ secrets.TF_VAR_GITHUB_ORG }}
```
Comment (TheFynx) [inline .github/workflows/platform-check.yaml:11]:
```suggestion
  TF_VAR_github_org: ${{ secrets.TF_VAR_GITHUB_ORG }}
```

File: .github/workflows/platform-check.yaml [hunk 2]
```diff
@@ -4,6 +4,13 @@
...
+  TF_VAR_GITHUB_TOKEN:  ${{ secrets.TF_VAR_GITHUB_TOKEN }}
```
Comment (TheFynx) [inline .github/workflows/platform-check.yaml:12]:
```suggestion
  TF_VAR_github_token:  ${{ secrets.TF_VAR_GITHUB_TOKEN }}
```

File: .github/workflows/platform-release.yaml [hunk 3]
```diff
@@ -4,6 +4,13 @@
 # repo then mentioned above as they will be overwritten upon the
 # next sync.
 
+# NOTE: we are temporarily throwing the kitchen sink of variables
+# at the workflow until we come up with a better solution at a
+# future date.
+env:
+  TF_VAR_GITHUB_ORG: ${{ secrets.TF_VAR_GITHUB_ORG }}
```
Comment (TheFynx) [inline .github/workflows/platform-release.yaml:11]:
```suggestion
  TF_VAR_github_org: ${{ secrets.TF_VAR_GITHUB_ORG }}
```

File: .github/workflows/platform-release.yaml [hunk 4]
```diff
@@ -4,6 +4,13 @@
...
+  TF_VAR_GITHUB_TOKEN:  ${{ secrets.TF_VAR_GITHUB_TOKEN }}
```
Comment (TheFynx) [inline .github/workflows/platform-release.yaml:12]:
```suggestion
  TF_VAR_github_token:  ${{ secrets.TF_VAR_GITHUB_TOKEN }}
```

File: .github/workflows/platform-check.yaml (hunk 1 above)
Comment (TheFynx) [reply .github/workflows/platform-check.yaml:11]: We don't want the variables to fully uppercase in our terraform variables and with TF environment variables just the TF_VAR needs to be capitalized and the rest needs to match the variable case in terraform

**Review by dngalenj1 — APPROVED** [review]

**Review by cjharries-nexient — APPROVED** [review]

**Pull request merged**